presentation/src/main/java/com/kdjj/presentation/viewmodel/my/MyRecipeViewModel.kt

```diff
@@ -0,0 +1,141 @@
+                        _liveSortType.value = SortType.SORT_BY_TIME
+                        _liveRecipeItemSelected.value = null
+                    }
+                }
+                .onFailure {
+                    //TODO 데이터 로드에 실패했을 경우 처리
+                }
+        }
+    }
+
+    private fun fetchLocalFavoriteRecipeList(page: Int) {
+        viewModelScope.launch {
+            favoriteRecipeUseCase(FetchLocalFavoriteRecipeListRequest(page))
+                .onSuccess { favoriteRecipeList ->
+                    if (_liveRecipeItemList.value?.isNotEmpty() == true && _liveSortType.value == SortType.SORT_BY_FAVORITE) {
+                        val myRecipeList = favoriteRecipeList.map { MyRecipeItem.MyRecipe(it) }
+                        _liveRecipeItemList.value = _liveRecipeItemList.value?.plus(myRecipeList)
+                    } else {
+                        val myRecipeList = favoriteRecipeList.map { MyRecipeItem.MyRecipe(it) }
+                        _liveRecipeItemList.value = listOf(MyRecipeItem.PlusButton) + myRecipeList
+                        _liveSortType.value = SortType.SORT_BY_FAVORITE
+                        _liveRecipeItemSelected.value = null
+                    }
+                }
+                .onFailure {
+                    //TODO 데이터 로드에 실패했을 경우 처리
+                }
+        }
+    }
+
+    private fun fetchLocalTitleRecipeList(page: Int) {
+        viewModelScope.launch {
+            titleRecipeUseCase(FetchLocalTitleRecipeListRequest(page))
+                .onSuccess { titleRecipeList ->
+                    if (_liveRecipeItemList.value?.isNotEmpty() == true && _liveSortType.value == SortType.SORT_BY_NAME) {
+                        val myRecipeList = titleRecipeList.map { MyRecipeItem.MyRecipe(it) }
+                        _liveRecipeItemList.value = _liveRecipeItemList.value?.plus(myRecipeList)
+                    } else {
+                        val myRecipeList = titleRecipeList.map { MyRecipeItem.MyRecipe(it) }
+                        _liveRecipeItemList.value = listOf(MyRecipeItem.PlusButton) + myRecipeList
+                        _liveSortType.value = SortType.SORT_BY_NAME
+                        _liveRecipeItemSelected.value = null
+                    }
+                }
+                .onFailure {
+                    //TODO 데이터 로드에 실패했을 경우 처리
+                }
+        }
+    }
+
+    fun moveToRecipeEditorActivity() {
+        _liveAddRecipeHasPressed.value = Event(Unit)
+    }
+
+    fun moveToRecipeSearchFragment(){
+        _eventSearchIconClicked.value = Event(Unit)
+    }
+
+    fun setSortType(sortType: SortType) {
+        if (_liveSortType.value != sortType) {
+            when (sortType) {
+                SortType.SORT_BY_TIME -> fetchLocalLatestRecipeList(0)
+                SortType.SORT_BY_FAVORITE -> fetchLocalFavoriteRecipeList(0)
+                else -> fetchLocalTitleRecipeList(0)
```

Comment:
```suggestion
                SortType.SORT_BY_TITLE -> fetchLocalTitleRecipeList(0)
```

else 보다는 명시하는게 더 가독성이 좋을 것 같습니다!